bootstrapvz/common/tasks/grub.py

```diff
@@ -27,9 +27,9 @@ def run(cls, info):
 		grub_def = os.path.join(info.root, 'etc/default/grub')
 		sed_i(grub_def, '^#GRUB_TERMINAL=console', 'GRUB_TERMINAL=console')
 		sed_i(grub_def, '^GRUB_CMDLINE_LINUX_DEFAULT="quiet"',
-		                'GRUB_CMDLINE_LINUX_DEFAULT="console=hvc0"')
+		                'GRUB_CMDLINE_LINUX_DEFAULT="init=/bin/systemd console=hvc0 console=ttyS0"')
```

Comment:
Should send console to both, from what I have been told by EC2 team; but that could be wrong. Seems to be working for me now though.